dune_api_scripts/update_appdata_view.py

```diff
@@ -20,18 +20,17 @@ def refresh(dune: DuneAPI, query: DuneQuery):
     )
 
 
-class Environment(Enum):
-    """Enum for Deployment Environments"""
-
-    STAGING = "barn"
-    PRODUCTION = "prod"
+@dataclass
+class Environment:
+    """Dataclass for Deployment Environments"""
+    value: str
```

Comment:
I was kindof hoping we could just add a third enum value like `TEST = "test"` which is the default (when none is provided as a script argument). 

This gave us some security when it comes to not arbitrarily / accidentally creating several different variations of the same table. 

If some one runs the script now with random input data we will find that several different copies of this able will be created in Dune.

Reply:
Makes sense to not allow free text creation of views in Dune. I've replaced the current implementation with the `test` env variable you suggested.